modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -7,6 +7,16 @@
 
 You must build a container disk with a virtual machine image and push it to a container registry before it can used with a virtual machine. You can then either import the container disk into a PVC using a data volume and attach it to a virtual machine, or you can attach the container disk directly to a virtual machine as an ephemeral `containerDisk` volume.
 
+[NOTE]
+====
+The size of a disk image inside a container disk is limited by the maximum layer size of the registry where the container disk is hosted. For Red Hat Quay, the default limit is 20Gi. You can configure the `MAXIMUM_LAYER_SIZE` field in the Red Hat Quay configuration schema. For more details, see the schema for Red Hat Quay configuration in the link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Manage Red Hat Quay] documentation.
```

Comment:
Looking into [the latest Quay docs](https://access.redhat.com/documentation/en-us/red_hat_quay/3.6/html-single/manage_red_hat_quay/index#quay-schema), the "schema" section is very short and just mentions a `config.yaml` file. I think we should also mention that file here to make it more explicit. I do think it's a good idea to link to the Quay docs, though. I'd prefer the link embedded into an existing sentence (see my example) but that is up to you.

How about this?

```
[NOTE]
====
The size of a disk image inside a container disk is limited by the maximum layer size of the registry where the container disk is hosted. For link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Red Hat Quay], the default limit is `20Gi`. You can change this value by editing the `MAXIMUM_LAYER_SIZE` field in the `config.yaml` file that is created when Red Hat Quay is first deployed.
====
```

Side question: should it be `20G` as it is shown in https://github.com/quay/quay/blob/master/config.py#L594 ?

Reply:
My only reservation is that if the file name changes, we are in trouble. Granted that isn't likely, but I just don't like to cross that line of assumption into another product's doc. The 20G question is a good example of that.... perhaps we should not even be repeating Quay documentation specifics in OpenShift doc.

@fabiand what do you think?

Reply:
I agree that we should not repeat other product's docs Thus +1 on removing the redundant parts and simply referring to the quay documentation.

Reply:
Changed to: 

`[NOTE]
====
The size of a disk image inside a container disk is limited by the maximum layer size of the registry where the container disk is hosted. See the link:https://access.redhat.com/documentation/en-us/red_hat_quay/[Red Hat Quay documentation] for information about the default layer size of the registry and how you can change this value by editing the appropriate field in the YAML configuration file that is created when Red Hat Quay is first deployed.
====`